src/main/java/com/github/zly2006/enclosure/config/Common.java

```diff
@@ -32,6 +32,8 @@ public class Common {
     public boolean showMessagePrefix = false;
     @SerializedName("teleport_cooldown")
     public int teleportCooldown = 10000;
+    @SerializedName("use_actionbar_message")
```

Comment:
`use_actionbar_welcome_message` may be better

Uou can also rename `allow_rich_message` to `allow_rich_welcome_message` like this

Reply:
Actually, this message inclues "ENTER" and "LEAVE", keep it consistency.